content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -90,44 +106,57 @@ algorithmSource:
 ...
 
 disablePreemption: true
-
 ```
 
-Although preemption of the scheduler is enabled by default, it is disabled if `PodPriority`
-feature is disabled.
+### Start an older version of Rescheduler in the cluster
+
+When priority or preemption is disabled, we must run Rescheduler v0.3.1 (instead
+of v0.4.0) to ensure that critical Pods are scheduled when nodes or cluster are
+under resource pressure. Since critical Pod annotation is still supported in
+this release, running Rescheduler should be enough and no other changes to the
+configuration of Pods should be needed.
```

Comment:
Should we talk about kubelet preemption logic here as it may cause surprises to user?

Reply:
SGTM